### DISCUSSION TITLE
Laravel Homestead on Lima-VM using Apple Silicon M1, is it possible?

### DISCUSSION THREAD

**Discussion by pathros** [lima-vm/lima#1336] [Q&A]
Does anybody know how to install [Laravel/Homestead](https://github.com/laravel/homestead) on Apple Silicon M1 using Lima-VM?

**Comment by afbjorklund** [comment]
Seems like it uses Vagrant, so it (and the image it uses) would need to be "ported" to lima

* https://github.com/lima-vm/lima/discussions/1271
